README.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-"
+)
+
+# just to keep the code clean below
+suppressWarnings(suppressPackageStartupMessages(library(dplyr)))
+```
+
+# Scales <img src="man/figures/logo.png" align="right" />
+
+[![Build Status](https://travis-ci.org/r-lib/scales.svg?branch=master)](https://travis-ci.org/r-lib/scales)
+[![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/scales/master.svg)](https://codecov.io/github/r-lib/scales?branch=master)
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/scales)](https://cran.r-project.org/package=scales)
+
+One of the most difficult parts of any graphics package is scaling, converting from data values to perceptual properties. The inverse of scaling, making guides (legends and axes) that can be used to read the graph, is often even harder! The idea of the `scales` package is to implement scales in a way that is graphics system agnostic, so that everyone can benefit by pooling knowledge and resources about this tricky topic.
```

Comment:
I think this should be reformulated to focussed on what most people actually care about — customisation ggplot2 scales

(And then just mention briefly that scales is a low-dependency package, and easily used by other graphics packages)